packages/client-axios/src/index.ts

```diff
@@ -5,6 +5,15 @@ import { createDefaultConfig, getUrl, mergeHeaders } from './utils';
 
 let globalConfig = createDefaultConfig();
 
+export class OpenApiAxiosError extends AxiosError {
+  constructor(e: AxiosError) {
+    super(e.message, e.code, e.config, e.request, e.response);
+    this.error = e.response?.data ?? {};
+  }
+
+  error: any;
+}
```

Comment:
You can move the content to types.ts or a new file, but it might be more convenient to keep it in the same file for readability. However, this is just a subjective opinion, and you are welcome to modify it based on the maintainer's preference.